tests/integration/testdata/publish/template_create_app_version.yaml

```diff
@@ -0,0 +1,38 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'sam-app
+
+  Sample SAM Template for sam-app
+
+  '
+Globals:
+  Function:
+    Timeout: 3
+Metadata:
+  AWS::ServerlessRepo::Application:
+    Author: user
+    Description: description
+    HomePageUrl: https://github.com/test/test
+    Labels:
+    - test-app
+    LicenseUrl: s3://<bucket-name>/LICENSE
```

Comment:
So the integ tests assume the SAM template has been packaged? What if it hasn't? What does the command do? Should we have tests to verify the expected behavior?

Reply:
I added exception handling for using not packaged template and unit/integration tests.